docs/config/enabling-control-service.rst

```diff
@@ -4,6 +4,8 @@
 Enabling the Flocker control service 
 ====================================
 
+The control service is the brain of Flocker; it can live anywhere in your cluster, and enabling it is an essential step in setting up your cluster.
```

Comment:
If you wanted to go into any more detail, you could also say that the control service is what serves the API for the cluster and you could say that all of the convergence agents coordinate their activities by communicating with the control service.

Also I think there's an extra comma in this sentence.  And if you want, you could put a newline after the `;` (making the lines a bit shorter).